dbt/adapters/athena/query_headers.py

```diff
@@ -0,0 +1,27 @@
+import dbt.adapters.base.query_headers
+
+class _QueryComment(dbt.adapters.base.query_headers._QueryComment):
+    """
+    Athena DDL does not always respect /* ... */ block quotations. 
+    This function is the same as _QueryComment.add except that 
+    a leading "-- " is prepended to the query_comment and any newlines
+    in the query_comment are replaced with " ". This allows the default
+    query_comment to be added to `create external table` statements.
+    """
+    def add(self, sql: str) -> str:
+        if not self.query_comment:
+            return sql
+
+        if self.append:
+            # replace last ';' with '<comment>;'
+            sql = sql.rstrip()
+            if sql[-1] == ";":
+                sql = sql[:-1]
+                return "{}\n-- /* {} */;".format(sql, self.query_comment.strip().replace("\n", " "))
```

Comment:
```suggestion
                return f"{sql}\n-- /* {self.query_comment.strip().replace("\n", " ")} */;"
```
How about using an f-string?

Reply:
thanks for the review @divy9393 - I like f strings too.

But, I've ripped this part of the code directly from [_QueryComment.add](https://github.com/dbt-labs/dbt-core/blob/43e3fc22c4eae4d3d901faba05e33c40f1f1dc5a/core/dbt/adapters/base/query_headers.py#L32-L45). I wouldn't want to modify it, so that 'future generations' have an easier time in the case something changes in dbt-labs/dbt-core